vagrant/provisioning/roles/haproxy/tasks/main.yml

```diff
@@ -88,7 +88,6 @@
       
       frontend main
         mode http
-        bind *:80
```

Comment:
do we need to bind to port 80 in order to auto-redirect to port 443 in case anybody uses http instead of https?

Reply:
@kikomanaskov333 , I still think we need to keep the bind to port 80, in order to send a redirect to port 443.

Reply:
@david-oc-miller , yes you're right, but then we need to change httpd default config (httpd.conf) not to bind to port 80.

Reply:
@kikomanaskov333 , yes, let's change the httpd bindings, so we know that only haproxy manages the ports that are visible to outside.  The httpd ports should not be open externally but instead should only be reachable through haproxy.